From: Markus Koschany <apo@gambaru.de>
Date: Mon, 1 Oct 2012 20:27:12 +0200
Subject: byzanz-playback-man-page-fix-wrong-synopsis

Apply patch from Jari Aalto again and fix the wrong command
in the synopsis.
---
 src/byzanz-playback.1 |    9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/src/byzanz-playback.1 b/src/byzanz-playback.1
index 28eb8b0..2a77f9c 100644
--- a/src/byzanz-playback.1
+++ b/src/byzanz-playback.1
@@ -1,4 +1,4 @@
-.\" Manual page for byzanz-record.
+.\" Manual page for byzanz-playback.
 .\" Written by Benjamin Otte <otte@gnome.org>
 .\"
 .\" Copyright (C) 2010 Benjamin Otte
@@ -16,20 +16,19 @@
 .SH NAME
 byzanz-playback \- Process a byzanz debug recording
 .SH SYNOPSIS
-.B byzanz-record
+.B byzanz-playback
 .RI [ options ] " INFILE OUTFILE"
 .SH DESCRIPTION
 Byzanz debug recording can be created by using the extension \fI.byzanz\fP.
 To convert these recordings into the other formats supported by Byzanz,
-\fBbyzanz-playback\fP can be used. The INFILE must e a byzanz debug file,
+\fBbyzanz-playback\fP can be used. The INFILE must be a byzanz debug file,
 the OUTFILE is the file to convert it to. Its extension determines the
 format to be used. See the \fBbyzanz-record\fP(1) man page for a list of
 supported formats and their extensions.
 .SH OPTIONS
-.SS "Help Options:"
 .TP
 \fB\-h\fR, \fB\-\-help\fR
-Show help options
+Show brief help.
 .SH SEE ALSO
 \fBbyzanz-record\fR(1)
 .SH AUTHOR
